apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -208,11 +208,9 @@ export class RushConfiguration {
 
   public static loadFromDefaultLocation(): RushConfiguration {
     const rushJsonLocation: string | undefined = RushConfiguration.tryFindRushJsonLocation();
-    if (rushJsonLocation) {
-      return RushConfiguration.loadFromConfigurationFile(rushJsonLocation);
-    } else {
-      throw new Error('Unable to find rush.json configuration file');
-    }
+    return rushJsonLocation
+      ? RushConfiguration.loadFromConfigurationFile(rushJsonLocation)
+      : Utilities.throwRushConfigNotFoundError();
```

Comment:
Make this more conventional...

Reply:
```
x = load();
if (!x) {
  throw
}
return x;
```

---
In reply to: [199306999](https://github.com/Microsoft/web-build-tools/pull/731#discussion_r199306999) [](ancestors = 199306999)